packit-ci.fmf

```diff
@@ -14,14 +14,14 @@
   prepare:
    - how: shell
      script:
-      - ln -s $(pwd) /var/tmp/rust-keylime_sources
+#      - ln -s $(pwd) /var/tmp/rust-keylime_sources
```

Comment:
leftover?

Reply:
No, it will be dropped after WIP is removed.